README.md

```diff
@@ -5,16 +5,15 @@
 
 [![DOI](https://zenodo.org/badge/452699400.svg)](https://zenodo.org/badge/latestdoi/452699400)
 ### Data
+Data is hosted at google drive: https://drive.google.com/drive/folders/1IwJjwSKpXYWWALKXM8nghqmlSvyu3BYq
 
-Data is available via [DVC](https://dvc.org/) and can be obtained by running  following commands within repo root directory:
+You can either:
+- manually download and extract it
+- use script `get_data.sh`
+  - create dedicated environment
```

Comment:
if we merge packages required to download data into the hydramp package, we could get rid of this. Plus, there is only one package required for downloading , so dedicated environment might be a bit too much :))

Reply:
as requsted I move `gdown` - lib for data download to `setup.py`